client_tipc.c

```diff
@@ -1164,6 +1288,13 @@ static int send_result(struct storage_client_session *session,
 	return send_response(session, result, msg, NULL, 0);
 }
 
+static int is_keymaster_ta(uuid_t *uuid)
+{
+	uuid_t keymaster_uuid = {0x5f902ace, 0x5e5c, 0x4cd8,
```

Comment:
can you please add comments here? I remembered that there is a header file that has all UUID definitions together. then you can add a note saying, this uuid below must be the same with the one defined in .../../xxx.h